### PR TITLE
Create pulp-fiction

### PR DESCRIPTION
**Pull request by makkoncept**

### CODE REVIEW

File: pulp-fiction [hunk 1]
```diff
@@ -0,0 +1 @@
+best of Tarantino?
```
Comment (cold-hotdog) [inline pulp-fiction:1]: first

**Review by cold-hotdog — COMMENTED** [review]
second

File: pulp-fiction (hunk 1 above)
Comment (ghost) [inline pulp-fiction:1]: first from mayank-hr

**Review by ghost — COMMENTED** [review]
second from mayank-hr

**Review by cold-hotdog — COMMENTED** [review]
third comment from cold-hotdog

**Comment by cold-hotdog** [comment]
fourth comment from cold-hotdog

**Comment by cold-hotdog** [comment]
fifth comment from cold-hotdog

**Comment by cold-hotdog** [comment]
sixth

**Comment by cold-hotdog** [comment]
7th

**Review by cold-hotdog — COMMENTED** [review]
8th

**Comment by cold-hotdog** [comment]
9th

**Comment by cold-hotdog** [comment]
10th

**Comment by cold-hotdog** [comment]
11th

**Review by cold-hotdog — COMMENTED** [review]
comment

**Review by cold-hotdog — APPROVED** [review]
c

**Review by cold-hotdog — APPROVED** [review]

**Review by cold-hotdog — COMMENTED** [review]
c

**Review by ghost — COMMENTED** [review]
comment

**Review by cold-hotdog — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**